docs/src/literate-tutorials/computational_homogenization.jl

```diff
@@ -193,6 +193,21 @@ using Test #src
 # the [`FerriteGmsh`](https://github.com/Ferrite-FEM/FerriteGmsh.jl) package:
 
 using FerriteGmsh
+
+# Overload for specific elements in this tutorial.
```

Comment:
My plan was to make a pr which will support both 0.3 and master beforehand instead, if you agree. Since it is nice with a FerriteGmsh that works with both, and avoid having to checkout FerriteGmsh.jl with a specific version for working with Ferrite#master.